lib/components/viewers/route-details.tsx

```diff
@@ -241,6 +198,26 @@ class RouteDetails extends Component<Props> {
             </StopContainer>
           </>
         )}
+        <LogoLinkContainer>
```

Comment:
Is there any way we can hide the shadow if there is no overflow? It looks a bit awkward as it is now